### ISSUE TITLE
Stuck on splash screen on Android 10 (v1.12.9)

### ISSUE THREAD

**Issue by bluesaxman** [avluis/Hentoid#693]
**Hentoid Version**: v1.12.9

**Device/Android Version**: Moto G7 (river) / LinageOS 17.1 (Android 10)

**Issue details / Repro steps**:
I installed the app from the git release,
I enabled storage permission for it,
Then I opened the app,
It then sits at the splash screen and never goes any farther. Its not frozen, as I can rotate the screen and it will update the splash message, but it never loads anything else.

**Stack trace / LogCat**: I'm not sure how to obtain this. 

@AVnetWS/admin-team

**Comment by RobbWatershed** [comment]
Thanks for your feedback. Which was the latest version that worked for you?

**Comment by RobbWatershed** [comment]
I haven't found anything suspicious in the logs for a Moto G7 recently.
My guess would be that something we do doesn't appeal to LineageOS.

You can get the logcat by following these steps :
1. install adb, use this instruction https://www.xda-developers.com/install-adb-windows-macos-linux/
2. after that just this command `adb logcat | findstr com.example.package` if hentoid, change `com.example.package` to `me.devsaki.hentoid`

**Comment by RobbWatershed** [comment]
No news for 1.5 months -> closed

**Comment by bluesaxman** [comment]
Sorry about that, I was moving house and didn't have an opertunity to contact back, this can stay closed, after figuring out how to use logcat, and subsequently breaking system permissions (which for some reason were already broken) I had to factory wipe, which fixed the problem. If you are curious, the problem stemmed from incorrect permissions and directory structure on the internal storage. Not with the app itself. It was hanging on the splash screen because it was unable to create the files it needed to in internal storage on first start. Several other apps were having the same issue, I manually created their folders and all apps including this one worked... until I installed an OTA and my phone wouldn't even boot. After getting everything set back up, no issues with this or other apps.

tl;dr, it was a me problem, not a you problem.

**Comment by RobbWatershed** [comment]
Oh, thanks for replying. Glad to know you finally got it working !
